<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEventDialog</class>
 <widget class="QDialog" name="NewEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBoxObj1">
     <property name="title">
      <string>Objeto 01</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidgetObject1">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxRel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Relações</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Associação:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBoxAttrib"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelCondAction">
        <property name="text">
         <string>Condição:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBoxCondit"/>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBoxNegate">
        <property name="text">
         <string>Negativa</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBoxObj2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Objeto 02</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidgetObject2">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBoxVal">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Valor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxNumber">
        <property name="text">
         <string>Numero:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="spinBoxNumber">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxVariable">
        <property name="text">
         <string>Variavel:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxVariable"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxFieldObject02">
        <property name="text">
         <string>Campo do Objeto 02:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QComboBox" name="comboBoxFieldObject2"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxSpecial">
        <property name="text">
         <string>Especial:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QComboBox" name="comboBoxSpecial"/>
      </item>
      <item row="7" column="0">
       <widget class="QDial" name="dialDirection"/>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="pushButtonChoose">
        <property name="text">
         <string>Escolher</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
